packages/web-components/src/components/ic-typography/ic-typography.tsx

```diff
@@ -203,7 +203,7 @@ export class Typography {
             onBlur={this.truncButtonBlur}
             onMouseDown={this.truncButtonFocusFromMouse}
             onClick={this.toggleExpanded}
-            aria-hidden="true"
+            tabindex="0"
```

Comment:
I don't think that explicitly setting `tabindex` to `0` is required, as a `<button>` will be focusable by default. So hopefully just removing aria-hidden is all that is required. The unit tests will fail as a result, so the snapshot files should be updated to reflect the change

Reply:
@ad9242 which npm command updates snapshots, and snapshots only?

Reply:
I don't think there is an npm command for it, but you can delete the existing snapshot file & then run the tests locally (npm run test), to have it regenerate the file and then validate manually that the changes are what you would expect